data/fwupd.service.in

```diff
@@ -6,6 +6,7 @@ Before=display-manager.service
 ConditionVirtualization=!container
 
 [Service]
+Environment="FWUPD_SYSCALL_FILTER=systemd"
```

Comment:
Is there no better way to tell you're running under systemd?

Reply:
https://stackoverflow.com/questions/39368185/how-can-a-program-detect-if-it-is-running-as-a-systemd-daemon perhaps

Reply:
The only one that looked reliable was the `parent process id` but even that feels like a hack. The env variable also means we can test it locally....

Reply:
Can we not use invocation id?

Reply:
Nah, that's set in the session too

Reply:
How about both parent process ID and env variable for local testing?

Reply:
Seems to work, fixed.

Reply:
Actually, I went back to the env var; `ppid()==1` doesn't have to be systemd -- it could be init.d, upstart or launchd.